### PR TITLE
Add Epinio installation on Minikube

### PR DESCRIPTION
**Pull request by juadk**
This PR adds documentation for installing Epinio on Minikube with different drivers.
I tested the deployment of the sample-app on the following drivers:
- kvm :heavy_check_mark: 
- docker :heavy_check_mark: 
- virtualbox :heavy_check_mark:

### CODE REVIEW

File: src/installation/install_epinio_on_minikube.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Epinio installation was tested on Minikube with the following drivers: docker, kvm, virtualbox.
+
+Specify the driver you want to use with the `--driver` option.
+
+```bash
+$ minikube start --driver=docker
+```
+
+Once the cluster ready, you need to install the metallb addon:
```
Comment (ldevulder) [inline src/installation/install_epinio_on_minikube.md:19]: More **IS** ready IMO.

File: src/installation/install_epinio_on_minikube.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Then, you have to configure metallb by giving it an IP address range.</br>
+For instance check your minikube IP with `minikube ip` and choose how many IP addresses you need.
+
+```bash
+$ minikube ip
+192.168.49.2
+```
+
+As an example, I choose the range `192.168.49.100-192.168.49.120`.</br>
```
Comment (ldevulder) [inline src/installation/install_epinio_on_minikube.md:33]: **we** choose instead of **I**, more "we" as a team, WDYT?

**Review by ldevulder — APPROVED** [review]
LGTM, just 2 small comments.

**Review by kkaempf — APPROVED** [review]
Verified working nicely on minikube 1.23 (k8s 1.21.5) and driver kvm. Thank you !

**Pull request merged**